{
    "Modified": "2020-09-10 01:15:00",
    "Published": "2018-08-16 20:29:00",
    "access": {
        "authentication": "NONE",
        "complexity": "MEDIUM",
        "vector": "NETWORK"
    },
    "assigner": "cve@mitre.org",
    "cvss": 4.3,
    "cvss-time": "2020-09-10 01:15:00",
    "cvss-vector": "AV:N/AC:M/Au:N/C:N/I:N/A:P",
    "cwe": "CWE-835",
    "id": "CVE-2018-14567",
    "impact": {
        "availability": "PARTIAL",
        "confidentiality": "NONE",
        "integrity": "NONE"
    },
    "last-modified": {
        "$date": 1599700500000
    },
    "references": [
        "http://www.securityfocus.com/bid/105198",
        "https://gitlab.gnome.org/GNOME/libxml2/commit/2240fbf5912054af025fb6e01e26375100275e74",
        "https://lists.debian.org/debian-lts-announce/2018/09/msg00035.html",
        "https://lists.debian.org/debian-lts-announce/2020/09/msg00009.html",
        "https://usn.ubuntu.com/3739-1/"
    ],
    "summary": "libxml2 2.9.8, if --with-lzma is used, allows remote attackers to cause a denial of service (infinite loop) via a crafted XML file that triggers LZMA_MEMLIMIT_ERROR, as demonstrated by xmllint, a different vulnerability than CVE-2015-8035 and CVE-2018-9251.",
    "vulnerable_configuration": [
        "cpe:2.3:a:xmlsoft:libxml2:2.9.8:*:*:*:*:*:*:*",
        "cpe:2.3:o:canonical:ubuntu_linux:14.04:*:*:*:lts:*:*:*",
        "cpe:2.3:o:canonical:ubuntu_linux:16.04:*:*:*:lts:*:*:*",
        "cpe:2.3:o:canonical:ubuntu_linux:18.04:*:*:*:lts:*:*:*",
        "cpe:2.3:o:debian:debian_linux:8.0:*:*:*:*:*:*:*"
    ],
    "vulnerable_configuration_cpe_2_2": [],
    "vulnerable_product": [
        "cpe:2.3:a:xmlsoft:libxml2:2.9.8:*:*:*:*:*:*:*",
        "cpe:2.3:o:canonical:ubuntu_linux:14.04:*:*:*:lts:*:*:*",
        "cpe:2.3:o:canonical:ubuntu_linux:16.04:*:*:*:lts:*:*:*",
        "cpe:2.3:o:canonical:ubuntu_linux:18.04:*:*:*:lts:*:*:*",
        "cpe:2.3:o:debian:debian_linux:8.0:*:*:*:*:*:*:*"
    ]
}